frontend/WikiContrib-Frontend/src/query.js

```diff
@@ -771,13 +773,10 @@ export class Query extends Component {
                                             )
                                           }
                                         />
-                                    </label>
                                       </Table.Cell>
                                       <Table.Cell
                                         style={{ textAlign: 'center' }}
                                       >
-                                        <label>
-                                          <div className="input_label">Gerrit Username</div>
```

Comment:
Placeholder text here.. n in name in small letter, u in username in small letter...